### ISSUE TITLE
A duplicated tab is placed at the END of vertical tab list

### ISSUE THREAD

**Issue by reikred** [piroor/treestyletab#835]
When I use a  "duplicate tab" menu, the duplicated tab is placed at the end of the TST vertical tab listing, rather than next to the tab that got duplicated.

Would it not be more logical if one could choose to open the duplicate tab next to (or under or over) the original? Or maybe even better, at the end of the subtree that is rooted  at the tab being duplicated?

I'm using https://addons.mozilla.org/en-US/firefox/addon/duplicate-in-tab-context-menu/ to provide the  the tab duplication function.Please correct me if that is the wrong way to do duplication when also using TST. I did some searching on the topic of TST and duplicate tab without luck.

**Comment by chronometric** [comment]
I think I know what the problem is:

In stock FF, without TST (and in the other popular browsers I've used), **middle-clicking the page reload button will duplicate the current tab**. By default, duplicates open immediately to the right. In TST, the default is just below and on the same level.

In the Addons settings for TST, the **Tabs** tab contains settings that allow you to implement the positionings you describe, for new tabs and duplicated. It takes a while to make tabs appear in a way that makes sense in complicated trees, but briefly:
* *Independent tab* will open at the bottom of all trees, on the first indentation level
* *Child of the current/source tab* opens indented one level and just below (top of child tree; can be changed to bottom of child tree in TST *Tree* options tab
* *A sibling of the current/source* means it will open at the highest point of the current tree on the same indentation level...unless it is a top-level tab, in which case it will open at the bottom of all tabs
* *Next sibling of the current/source tab* opens tabs just below and on the same level.

Now, when I installed the Duplicate in Tab Context Menu addon along with TST, it apparently ignores the settings set by TST, as well as the FF default, and will only open its duplicated tabs at the bottom. Thankfully the TST middle-click options are unaffected.

I'd suggest ditching the context-menu duplicator addon; it has no useful settings and could only conceivably be useful if your mouse doesn't have a middle button. And in that case, I would suggest a more complex manager called *Tab Utilities,^ which can set the function of mouse buttons and mouse-keyboard combs when clicked in different places on the tab bar.
